### ISSUE TITLE
https://opensource.google/

### ISSUE THREAD

**Issue by werymanen** [darkreader/darkreader#2978]
**Website**
https://opensource.google/

**How to Reproduce**
An example of this is
> 1. Go to https://opensource.google/
> 3. See that the logo on the top left is scaled

**Expected behavior**
The logo should not scale, should stay the same

**Actual behavior**
The logo scales, making parts of it invisible

**Screenshots**
Enabled: 
![logo](https://user-images.githubusercontent.com/49951816/86126803-35203d00-badf-11ea-9d2b-40d744fea78f.png)
Disabled: 
![logo-correct](https://user-images.githubusercontent.com/49951816/86126821-3c474b00-badf-11ea-9c2c-ceac2e7c9d27.png)

**System info:**
 - OS: Linux
 - Browser: Brave (Chromium-based)
 - Darkreader Version: 4.9.13

**Comment by Gusted** [comment]
Not sure what happend their... It seems we scaled?

**Comment by GalaxySH** [comment]
Looks fine on Firefox (desktop), though it should be inverted.

**Comment by sprinkledcheese** [comment]
Looks like a blob, typical dark reader issue on chrome.

**Comment by sprinkledcheese** [comment]
It is a blob: 
![chrome_BTDTjaXZ3w](https://user-images.githubusercontent.com/65959962/87140149-a79ed300-c2be-11ea-9cfa-d78747aeaba0.png)

**Comment by Gusted** [comment]
Yeah it was detected as `white-ish` image than we simply darken it and convert to a blob. But it got resized for some reason?

**Comment by I3HowA** [comment]
I've noticed similar issues on other websites too

Website1:
[https://adguard-vpn.com/en/welcome.html](https://adguard-vpn.com/en/welcome.html)
DarkReader ON:
![Screen Shot 2020-07-11 at 22 47 50 PM](https://user-images.githubusercontent.com/48854923/87239843-98a44780-c3c8-11ea-9e3a-dcd6342a8e8d.png)
DarkReader OFF:
![Screen Shot 2020-07-11 at 22 48 51 PM](https://user-images.githubusercontent.com/48854923/87239859-c12c4180-c3c8-11ea-937d-508830929922.png)

Website2:
[https://www.nytimes.com/wirecutter/](https://www.nytimes.com/wirecutter/)
DarkReader ON:
![Screen Shot 2020-07-11 at 22 51 48 PM](https://user-images.githubusercontent.com/48854923/87239926-6515ed00-c3c9-11ea-91d7-255eece61115.png)
DarkReader OFF:
![Screen Shot 2020-07-11 at 22 54 28 PM](https://user-images.githubusercontent.com/48854923/87239939-8545ac00-c3c9-11ea-96f0-6e3c2d2c2788.png)

System info:
OS: MacOS 10.14.4
Browser: Chrome 83.0.4103.116
Darkreader Version: 4.9.15

**Comment by Gusted** [comment]
I will pass the info along and see where the bug relays.
